build.sc

```
@@ -1417,15 +1424,17 @@ object dev extends MillPublishScalaModule {
       case wd0 +: rest =>
         val wd = os.Path(wd0, T.workspace)
         os.makeDir.all(wd)
-        try Jvm.runSubprocess(
+        try {
+          Jvm.runSubprocess(
             Seq(launcher().path.toString) ++ rest,
             forkEnv(),
             workingDir = wd
           )
-        catch {
-          case e: Throwable => () /*ignore to avoid confusing stacktrace and error messages*/
```

This hides any failures as `dev.run` will succeed.